packages/phrases/src/locales/en/translation/admin-console/application-details.ts

```diff
@@ -4,14 +4,14 @@ const application_details = {
   settings: 'Settings',
   settings_description:
     'Applications are used to identify your applications in Logto for OIDC, sign-in experience, audit logs, etc.',
-  advanced_settings: 'Advanced settings',
```

Comment:
I think we delibrately use sentence case in these field labels?

Reply:
There is a specific linear issue for this, update those terms using capitalized case